app/javascript/packs/applications.js

```diff
@@ -1,7 +1,34 @@
-import { initialize as planSelector } from 'Applications/plan_selector'
-import { CreateApplication } from 'Applications/create_application'
+import { ApplicationFormWrapper as renderApplicationForm } from 'Applications/ApplicationForm'
+import { safeFromJsonString } from 'utilities/json-utils'
 
 document.addEventListener('DOMContentLoaded', () => {
-  planSelector()
-  window.createApplication = new CreateApplication()
+  // Remove formtastic forms
+  document.getElementById('cinstance_service_plan_id_input').remove()
+  document.getElementById('cinstance_plan_input').remove()
+
+  const form = document.querySelector('.inputs > ol')
+  const tempDiv = document.createElement('div')
+  form.prepend(tempDiv)
+
+  const { dataset } = document.getElementById('metadata-form')
+
+  const plans = safeFromJsonString(dataset.application_plans)
+  const servicesContracted = safeFromJsonString(dataset.services_contracted)
+  const relationServiceAndServicePlans = safeFromJsonString(dataset.relation_service_and_service_plans)
```

Comment:
Here's where it fails what I've commented above... Also, will fail with the rest  if there are not escaped properly.